docs/guide/docs/channels/converse.md

```diff
@@ -3,76 +3,189 @@ id: converse
 title: Converse API
 ---
 
-The Converse API is an easy way to integrate Botpress with any application or any other channels. This API will allow you to speak to your bot and get an answer synchronously.
+The Converse API is an easy way to integrate Botpress with any application or any other channel.
 
-## Usage (Public API)
+## Public API
 
-`POST /api/v1/bots/{botId}/converse/{userId}` where **userId** is a unique string identifying a user that chats with your bot (**botId**), you can use random string for **userId**, also don't forget to set the `Content-Type` HTTP header to `application/json`.
+The Public API allows you to interact with your chatbot and get an answer synchronously. In other words, you can hold a 'conversation' with your chatbot using the converse API. The only difference is that responses are delivered to the calling application as JSON and not through a channel with a GUI.
 
-### Request Body
+### Request Header
+
+To 'speak' with your chatbot via the converse API, make a call with the HTTP header `Content-Type` set to `application/json`)
 
+### Request URL
+Make a post request to the converse API. below is an example of the request format
+```json
+POST <your-bot-url.com>/api/v1/bots/{botId}/converse/{userId}
+```
+ -Replace **userId** with any unique string to represent a user chatting with your chatbot (**botId**).
+ -For **botId**, go to `\data\bots\BOTNAME`, then open the `bot.config.json` file where the last two lines will specify bot id.
```

Comment:
I'm sorry, we use unix syntax pretty much everywhere, I don't see why we would switch to windows  \ now

```suggestion
 -For **botId**, go to `/data/bots/BOTNAME`, then open the `bot.config.json` file where the last two lines will specify bot id.
```